templates/workloads/partials/cf/efs-access-point.yml

```diff
@@ -0,0 +1,20 @@
+{{- if .Storage}}
+{{- if .Storage.AccessPoint}}
+AccessPoint:
+  Type: AWS::EFS::AccessPoint
+  Properties:
+    ClientToken: String
```

Comment:
Does the value for `ClientToken` here matter?

Reply:
Good catch. I'll pull it in my next PR.